openfisca_france/model/prelevements_obligatoires/impot_revenu/reductions_impot.py

```diff
@@ -2470,16 +2448,14 @@ class reduction_impot_exceptionnelle(DatedVariable):
     column = FloatCol(default = 0)
     entity_class = FoyersFiscaux
     label = u"Réduction d'impôt exceptionnelle"
-    url = "https://www.legifrance.gouv.fr/affichTexteArticle.do;jsessionid=2342AB1B1727908BFF5295A7F51A5A65.\
-        tpdjo13v_1?cidTexte=JORFTEXT000029349482&idArticle=LEGIARTI000029350526&dateTexte=20140809&categorieLien=cid"
 
     @dated_function(start = date(2013, 1, 1), stop = date(2013, 12, 31))
     def function(self, simulation, period):
         period = period.start.offset('first-of', 'year').period('year')
-        nb_adult = simulation.calculate('nb_adult', period)
-        nbptr = simulation.calculate('nbptr', period)
-        rfr = simulation.calculate('rfr', period)
-        params = simulation.legislation_at(period.start).ir.reductions_impots.reduction_impot_exceptionnelle
+        nb_adult = simulation.calculate('nb_adult')
+        nbptr = simulation.calculate('nbptr')
+        rfr = simulation.calculate('rfr')
```

Comment:
Why doesn't this one need a period ? I can understand for the two previous ones, but `rfr` changes every year, no ?